gno.land/cmd/gnoland/config_edit.go

```diff
@@ -0,0 +1,264 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"reflect"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/gnolang/gno/tm2/pkg/bft/config"
+	"github.com/gnolang/gno/tm2/pkg/bft/state/eventstore/types"
+	"github.com/gnolang/gno/tm2/pkg/commands"
+)
+
+var errInvalidConfigEditArgs = errors.New("invalid number of config edit arguments provided")
+
+type configEditCfg struct {
+	configPath string
+}
+
+// newConfigEditCmd creates the new config edit command
+func newConfigEditCmd(io commands.IO) *commands.Command {
+	cfg := &configEditCfg{}
+
+	cmd := commands.NewCommand(
+		commands.Metadata{
+			Name:       "edit",
+			ShortUsage: "config edit <key> <value>",
+			ShortHelp:  "Edits the Gno node configuration",
+			LongHelp: "Edits the Gno node configuration locally by setting " +
+				"the given value to the appropriate key",
+		},
+		cfg,
+		func(_ context.Context, args []string) error {
+			return execConfigEdit(cfg, io, args)
+		},
+	)
+
+	return cmd
+}
+
+func (c *configEditCfg) RegisterFlags(fs *flag.FlagSet) {
+	fs.StringVar(
+		&c.configPath,
+		"config-path",
+		"./config.toml",
+		"the path to the node's config.toml",
+	)
+}
+
+func execConfigEdit(cfg *configEditCfg, io commands.IO, args []string) error {
+	// Load the config
+	loadedCfg, err := config.LoadConfigFile(cfg.configPath)
+	if err != nil {
+		return fmt.Errorf("unable to load config, %w", err)
```

Comment:
This is just a personal preference; I use the `<message>, %w` wrap syntax across my repos and inside the gno codebase (see other commands, `toml`, `gnodev`...)

I'd prefer to keep it as is in this PR, but am open to having a linter for this sort of thing in a future PR, because it should be standardized, and because we don't have a codebase standard already for error wrapping

Reply:
adding to my infinite list of "nitpicky notes I may get to at one point"...